dali/test/python/test_operator_spectrogram.py

```diff
@@ -54,11 +56,9 @@ def define_graph(self):
 
     def iter_setup(self):
         data = self.iterator.next()
-        # randomly insert extra axis (channels?)
-        r = np.random.randint(-1, 2)
```

Comment:
This tested the changing ndim. We don't support that and now it causes an error, so I changed it to random but constant ndim.

Reply:
:+1: